src/main/webapp/login.jsp

```diff
@@ -41,6 +41,11 @@ if(request.getSession() != null)
 		
 	}
 }
+
+//Try talking to the database to force an early error if there's some db problem so we see error messages before trying to log in
+
+Connection conn = Database.getCoreConnection();
```

Comment:
Should probably close connections here immediately afterwards?

Reply:
You're right